signicat/Issues/kvkTradeRegister/description.xml

```diff
@@ -0,0 +1,463 @@
+				<nl>De inhoud van de niet gestructureerd op te nemen beperking in de volmacht</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="permissionToChangeTradeRegister">
+			<Name>
+				<en>Permission to file changes to the trade register</en>
+				<nl>Mag opgave handelsregister doen</nl>
+			</Name>
+			<Description>
+				<en>If the authorized representative has the permission to file changes to the trade register</en>
+				<nl>Indien de gemachtigde opgave mag doen aan het handelsregister</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="specialLegalSituationOfficial">
+			<Name>
+				<en>Special legal situation of the official</en>
+				<nl>Bijzondere rechtstoestand functionaris</nl>
+			</Name>
+			<Description>
+				<en>The special legal situation of the official, eg: Bankruptcy, debt restructuring or suspension of payment</en>
+				<nl>De bijzondere rechtstoestand van de functionaris, zoals: Faillissement, schuldsanering of surseance van betaling</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="restrictionInLegalActionOfficial">
+			<Name>
+				<en>Restriction in legal action of the official</en>
+				<nl>Beperking in rechtshandeling functionaris</nl>
+			</Name>
+			<Description>
+				<en>The restriction in legal action of the official, zoals: Provisional rule, placed under guardianship of placed under administration</en>
+				<nl>De beperking in rechtshandeling van de functionaris, zoals: Provisioneel bewind, onder curatele gesteld of onder bewind gesteld</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="suspensionStart">
+			<Name>
+				<en>Start date of suspension</en>
+				<nl>Aanvangsdatum van de schorsing</nl>
+			</Name>
+			<Description>
+				<en>Indicates when the official's suspension started</en>
+				<nl>Geeft aan wanneer de schorsing van de functionaris is gestart</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="suspensionEnd">
+			<Name>
+				<en>End date of suspension</en>
+				<nl>Einddatum van de schorsing</nl>
+			</Name>
+			<Description>
+				<en>Indicates when the official's suspension has ended</en>
+				<nl>Geeft aan wanneer de schorsing van de functionaris beeindigd is</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="emancipationOfMinor">
+			<Name>
+				<en>Emancipation of a minor</en>
+				<nl>Handlichting</nl>
+			</Name>
+			<Description>
+				<en>Emancipation of a minor</en>
+				<nl>Handlichting houdt in dat een minderjarige door toewijzing van een rechter dezelfde bevoegdheden toegekend krijgt als een meerderjarige.</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="hasRestriction">
+			<Name>
+				<en>Has restrictions</en>
+				<nl>Heeft beperking</nl>
+			</Name>
+			<Description>
+				<en>Indication of whether there is some kind of restriction that affects the determination of an authorization</en>
+				<nl>Indicatie of er sprake is van enige vorm van beperking, die van invloed is op het bepalen van een bevoegdheid ('Ja', 'Nee')</nl>
```

Comment:
de apostrof bij "ja" en "nee" kan weg zoals hierboven ook niet gedaan is? In iedergeval suggestie om recht te trekken :).